src/xrpld/app/misc/WasmVM.cpp

```diff
@@ -27,6 +27,53 @@
 
 namespace ripple {
 
+NotTEC
+preflightEscrowWasm(
```

Comment:
It duplicates all  code from the `runEscrowWasm`, what the point of creating it? Just call `runEscrowWasm`

Reply:
Different error codes